manifests/mod/remoteip.pp

```diff
@@ -0,0 +1,27 @@
+class apache::mod::remoteip (
+  $header            = 'X-Forwarded-For',
+  $proxy_ips         = [ '127.0.0.1' ],
+  $proxies_header    = undef,
+  $trusted_proxy_ips = undef,
```

Comment:
Actually, the pattern in the readme for apache::mod::* classes is just to have a * after the name if they have parameters, no further docs in the README, so this looks good!